compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -212,6 +212,28 @@ class CompilationTests {
     compileFilesInDir("tests/explicit-nulls/run", defaultOptions and "-Yexplicit-nulls")
   }.checkRuns()
 
+  // Flexible types tests
+  // @Test def flexibleTypesRun: Unit = {
+  //   implicit val testGroup: TestGroup = TestGroup("flexibleTypesRun")
+  //   compileFilesInDir("tests/flexible-types/run", flexibleTypesOptions)
+  // }.checkRuns()
+
+  @Test def flexibleTypesNeg: Unit = {
+    implicit val testGroup: TestGroup = TestGroup("flexibleTypesNeg")
+    aggregateTests(
+      // compileFilesInDir("tests/explicit-nulls/flexible-types/neg", defaultOptions and "-Yexplicit-nulls"),
+      compileFilesInDir("tests/explicit-nulls/flexible-types/common", defaultOptions and "-Yexplicit-nulls"),
```

Comment:
Should this be `explicitNullsOptions`?

Reply:
Oh, I forgot I added `explicitNullsOptions` back.